components/datetime/src/lib.rs

```diff
@@ -125,3 +124,25 @@ pub mod preferences {
     #[doc = "\n"] // prevent autoformatting
     pub use icu_locale_core::preferences::extensions::unicode::keywords::NumberingSystem;
 }
+
+/// Types that can be fed to [`DateTimeFormatter`]/[`FixedCalendarDateTimeFormatter`].
+pub mod input {
```

Comment:
issue: add `TimeZoneInfo` again

Reply:
Oh, I thought we didn't want it here. Okay.